packages/framework/src/Twig/ImageExtension.php

```diff
@@ -244,4 +264,59 @@ protected function getImageHtmlByEntityName(array $attributes, $entityName, $add
             'imageCssClass' => $this->getImageCssClass($entityName, $attributes['type'], $attributes['size']),
         ]);
     }
+
+    /**
+     * @param array $attributes
+     * @return bool
+     */
+    protected function shouldUseLazyLoad(array $attributes): bool
+    {
+        // no-image should not be lazy loaded
+        if ($attributes['src'] === $this->getEmptyImageUrl()) {
+            return false;
+        }
+
+        return array_key_exists('lazy', $attributes) ? (bool)$attributes['lazy'] : $this->isLazyLoadEnabled;
+    }
+
+    /**
+     * @param array $attributes
+     * @return array
+     */
+    protected function extractHtmlAttributesFromAttributes(array $attributes): array
+    {
+        $htmlAttributes = $attributes;
+
+        unset($htmlAttributes['type'], $htmlAttributes['size'], $htmlAttributes['lazy']);
```

Comment:
nitpick: I am thinking about declaring not html attributes in constant and then unsetting this in loop

Reply:
Good idea, changed in 73e9ce93a